### PR TITLE
Add testing guide

### PR DESCRIPTION
**Pull request by devinbileck**
In anticipation of our upcoming v0.9.0 release and use of TestQuality for managing and tracking the testing process, I have put together a basic testing guide.

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+First, some definitions within the context of TestQuality.
+
+- **Test Case:** a set of conditions under which a tester will determine whether the system under test satisfies requirements or works correctly.
+- **Test Suite:** a collection of test cases that are intended to be used to test the system to show that it has some specified set of behaviours.
+- **Test Plan:** defines a particular testing scope with testing activities.
+- **Test Plan Run:** an occurrence in which a particular test plan is executed.
+
+### Testing Workflow
+
+Once logged in to TestQuality, from the dashboard select the bisq project.
```
Comment (ripcurlx) [inline docs/testing.md]: `bisq` should be `Bisq`

**Review by ripcurlx — APPROVED** [review]
utACK - @devinbileck Thanks for putting together this document! Release testing starts today, as I will only add important bug fixes or minor UI fixes from now on. The release is planned to take place next Monday.

**Comment by HarryMacfinned** [comment]
@devinbileck , 
Sorry if I misunderstand,
but will the "internal" links in the <code>testing-guide.md</code> document be properly translated ?
I mean eg in the first lines: 
https://github.com/devinbileck/bisq/blob/testing-guide/docs/build.md ?
https://github.com/devinbileck/bisq/blob/testing-guide/docs/dev-setup.md ?
will it be translated in https://github.com/bisq-network/... ?

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Pull request merged**